docs/setup/k8s/running-in-production-k8s.md

```diff
@@ -62,50 +63,65 @@ Below is a quick example of how to setup the a specific stable version of nuclio
+If you select to handle the implementation yourself, follow these guidelines; the referenced configurations are all [Helm values](/hack/k8s/helm/nuclio/values.yaml):
 
-That being said, here are a few guidelines to get you on your way:
+- Set `*.image.repository` and `*.image.tag` to [freeze a qualified version](#version-freezing), and ensure that the configured images are accessible to the Kubernetes deployment.
+- Set `*.image.pullPolicy` to `Never` or to `IfNotPresent` to ensure that Kubernetes doesn't try to fetch the images from the web.
+- Set `offline` to `true` to put Nuclio in "offline" mode.
+- Set `dashboard.baseImagePullPolicy` to `Never`.
+- Set `registry.pushPullUrl` to a registry URL that's reachable from your system.
+- To use a Nuclio templates library (optional), set `dashboard.templatesArchiveAddress` to a location for serving the templates.
+  Note that you need to package the templates yourself and ensure that they're served locally in a way that's accessible to your system.
 
-- Most definitely [version freeze](#version-freezing). Also, set `*.image.pullPolicy` to `Never` or `IfNotPresent` to make sure k8s won't try to access the web to fetch images at any point in time.
-- Set `Values.offline=true` in the helm values, to put nuclio in "offline" mode. Set `dashboard.baseImagePullPolicy=Never`.
-- Needless to say, in this scenario, you will have to configure nuclio with `registry.pushPullUrl` which is reachable from your system.
-- The processor and onbuild images will also have to be accessible to the dashboard in your environment, as they are required for the building process - (by `docker build`, or [kaniko](#using-kaniko-as-an-image-builder)).
-  This can be tricky as you have to either make those images available to the k8s docker daemon or pull-able from a reachable registry, where they should be preloaded. Use `Values.registy.defaultBaseRegistryURL` to point nuclio at searching in your registry for those images, rather then at the default location of `quay.io/nuclio`.
-  To save some work on setting up a registry and preloading the onbuild images to it (or as a reference to what it should include) - take a look at the [prebaked-registry](https://github.com/nuclio/prebaked-registry).
-- For the Nuclio templates library to be available to you, you'll have to package that yourself, and have it served locally, somewhere within reach of your system. To point Nuclio to it, set `Values.dashboard.templatesArchiveAddress` to where you serve the templates.
+<a id="air-gapped-envir-processor-n-on-build-images"></a>In addition, the processor and on-build images must also be accessible to the dashboard in your environment, as they're required for the build process (either by `docker build` or [Kaniko](#using-kaniko-as-an-image-builder)).
```

Comment:
```suggestion
<a id="air-gapped-envir-processor-n-on-build-images"></a>In addition, the processor and onbuild images must also be accessible to the dashboard in your environment, as they're required for the build process (either by `docker build` or [Kaniko](#using-kaniko-as-an-image-builder)).
```
`onbuild` is a term from the code and is one word in the name of these images as well

Reply:
I changed it to _"onbuild"_ (with the quotation marks) in the two locations in which it was used in the text, as I saw this is what we did in other doc locations and in the UI and this is also what's done in some of the Docker and k8s docs — see commit 424d83c.

Reply:
np